src/core/rubberbandmodel.cpp

```diff
@@ -131,14 +131,14 @@ void RubberbandModel::removeVertices( int index, int count )
     return;
 
   mPointList.remove( index, count );
-  emit verticesRemoved( index, count );
-  emit vertexCountChanged();
 
   if ( mCurrentCoordinateIndex >= mPointList.size() )
   {
     setCurrentCoordinateIndex( mPointList.size() - 1 );
-    emit currentCoordinateChanged();
   }
+
+  emit verticesRemoved( index, count );
```

Comment:
I'm scared of this change :wink: did you try line tracking (both new line and resuming previous line) just to make sure we're not regressing on that front? Thanks

Reply:
Any way to simulate this on desktop?

Reply:
@suricactus , ah, I was hoping you'd ask :)

You can use the `test/nmea_server/nmeaserver.py` to emulate a nmea stream. For e.g.

python3 nmeaserver.py -t tcp -p 1111 TrimbleR1.txt

Reply:
@suricactus , you can also just take an APK here and run around with your phone on :)

Reply:
I assume this looks good. I don't see anything scary with continue tracking. 
![image](https://github.com/opengisch/QField/assets/2820439/093a6a29-e99a-4f39-9ac1-c84ec2015817)